testing/integrations_test.go

```diff
@@ -203,3 +203,26 @@ func baseGithubScenario() *scenariobuilder.TodocheckScenario {
 func baseGitlabScenario() *scenariobuilder.TodocheckScenario {
 	return baseGithubScenario()
 }
+
+func TestPublicYoutrackIntegration(t *testing.T) {
+	err := scenariobuilder.NewScenario().
+		OnlyRunOnCI().
+		WithBinary("../todocheck").
+		WithBasepath("./scenarios/integrations/youtrack_public_incloud").
+		WithAuthTokenFromEnv("TESTS_YOUTRACK_PUBLIC_INCLOUD_APITOKEN").
+		WithConfig("./test_configs/integrations/youtrack_public_incloud.yaml").
+		ExpectTodoErr(
+			scenariobuilder.NewTodoErr().
+				WithType(errors.TODOErrTypeIssueClosed).
+				WithLocation("scenarios/integrations/youtrack_public_incloud/main.go", 4).
+				ExpectLine("// TODO DEMO-20: A closed issue")).
+		ExpectTodoErr(
+			scenariobuilder.NewTodoErr().
+				WithType(errors.TODOErrTypeNonExistentIssue).
+				WithLocation("scenarios/integrations/youtrack_public_incloud/main.go", 5).
+				ExpectLine("// TODO DEMO-1234: a non-existent issue")).
```

Comment:
This todo scenario doesn't correspond to the actual file under test

Reply:
Should be fixed now.